static/app/views/projectDetail/projectScoreCards/projectStabilityScoreCard.spec.tsx

```diff
@@ -0,0 +1,151 @@
+import {Organization} from 'sentry-fixture/organization';
+
+import {render, screen} from 'sentry-test/reactTestingLibrary';
+
+import {SessionFieldWithOperation} from 'sentry/types';
+import ProjectStabilityScoreCard from 'sentry/views/projectDetail/projectScoreCards/projectStabilityScoreCard';
+
+describe('ProjectDetail > ProjectStability', function () {
+  let endpointMock: jest.Mock;
+  const organization = Organization();
+
+  const selection = {
+    projects: [1],
+    environments: [],
+    datetime: {
+      start: null,
+      end: null,
+      period: '14d',
+      utc: null,
+    },
+  };
+
+  afterEach(function () {
+    MockApiClient.clearMockResponses();
+  });
+
+  it('renders crash free users', async function () {
+    endpointMock = MockApiClient.addMockResponse({
+      url: `/organizations/${organization.slug}/sessions/`,
+      body: {
+        groups: [
+          {
+            totals: {
+              [SessionFieldWithOperation.CRASH_FREE_RATE_USERS]: 0.99,
+            },
+          },
+        ],
+      },
+      status: 200,
+    });
+
+    render(
+      <ProjectStabilityScoreCard
+        field={SessionFieldWithOperation.CRASH_FREE_RATE_USERS}
+        hasSessions
+        selection={selection}
+        isProjectStabilized
+        query="test-query"
+      />,
+      {
+        organization,
+      }
```

Comment:
by default we provide the default org object, if we're not adding/changing anything we can remove it
```suggestion
```

Reply:
thx for the hint! 🙌